README.md

```diff
@@ -1,82 +1,116 @@
+    <h1>ClickHouse Java Client & JDBC Driver</h1>
+</div>
+
+Table of Contents
+* [About The project](#about-the-project)
+* [Important Updates](#important-updates)
+* [Installation](#installation)
+* [Client V2](#client-v2)
+  * [Artifacts](#artifacts)
+  * [Features](#features)
+  * [Examples](#examples)
+* [Client V1](#client-v1)
+  * [Artifacts](#artifacts-1)
+  * [Features](#features-1)
+  * [Examples](#examples-1)
+* [Documentation](#documentation)
+* [Contributing](#contributing)
+
+## About the Project
+
+This is the official Java Client and JDBC for ClickHouse Database (https://github.com/ClickHouse/Clickhouse).
+Java client is the base component and has own API for working with ClickHouse in a "direct" way. JDBC driver is
+a library implementing JDBC API 1.3 on top of the Java client.
+
+Currently, there is a new version of the Java Client in developer preview. It is going to replace the old one very soon.
+Source code of the new version is located in the `client-v2` directory and usage examples are in `examples/client-v2` directory.
+
+## Important Updates
+
+Next components are deprecated and will be removed in the future releases:
+| Component                      | Version | Comment                                          |
 |--------------------------------|---------|--------------------------------------------------|
-| Clickhouse ClI Client package  | 0.7.0   | Use clickhouse-client directly instead           | 
-| ClickHouse GRPC Client package | 0.7.0   | Please use the clickhouse http protocol instead  |
-## Usage
+| Clickhouse ClI Client          | 0.7.0   |                                                  |
+| ClickHouse GRPC Client         | 0.7.0   | Please use the clickhouse http protocol instead  |
+
+## Installation
+
+Release versions are available in the Maven Central Repository. Nightly builds are available in the Sonatype Nexus Repository.
+
+Nightly Builds Repo: https://s01.oss.sonatype.org/content/repositories/snapshots/com/clickhouse/
+
+## Client V2
+
+### Artifacts
+
+| Component                 | Maven Central Link |
+|---------------------------|--------------------|
+| ClickHouse Java Client V2 | [![Maven Central](https://img.shields.io/maven-central/v/com.clickhouse/client-v2)](https://mvnrepository.com/artifact/com.clickhouse/client-v2) |
+
+### Compatibility
+
+| ClickHouse Version | Client Version | Comment                                                                                                                                          |
+|--------------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| Server >= 23.0     | 0.6.2          |                                                                                                                               |
```

Comment:
I'm not sure we can claim it. We run the tests against [the oldest LTS version 24.3](https://github.com/ClickHouse/clickhouse-java/blob/main/.github/workflows/analysis.yml#L31), which is aligned with [ClickHouse support policy](https://github.com/ClickHouse/ClickHouse/blob/master/SECURITY.md)